core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,45 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput') {
+        var choices = $scope.getAnswerChoices().length;
+        var answerGroups = responsesService.getAnswerGroups();
+        // First push all choice indices into an array
+        var choiceArray = [];
+        for (var i = 0; i < choices; i++) {
+          choiceArray.push(i);
+        }
+        /* Then cycle through answerGroup rule_specs
+        and put all of them into an array */
+        var answerArray = [];
```

Comment:
Maybe call this handledChoicesArray? I think it better conveys what the array is storing -- i.e., the choices which have been taken care of by the existing answer groups.